install_config/authentication/configuring_authentication_user_agent.adoc

```diff
@@ -0,0 +1,100 @@
+[[configuring-authentication-user-agent-api]]
+= Configuring User Agents to Prevent Access to the APIs
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+{product-title} implements a user agent that can be used to prevent an application developer's CLI accessing the {product-title} API.
+
+
+
+== Configuring User Agents
+
+User agents for the {product-title} CLI are constructed from a set of values
+within {product-title}:
+
+----
+<command>/<version> (<platform>/<architecture>) <client>/<git_commit>
+----
+
+For example: 
+
+* If you have the following values:
++
+** <command> = `oc`
+** <version> = The client version. For example, `v3.5.0`. Requests made against the Kubernetes
+API at `/api` receive the Kubernetes version, while requests made against the
+{product-title} API at `/oapi` receive the {product-title} version (as specified
+by `oc version`)
+** <platform> = `linux`
+** <architecture> = `amd64`
+** <client> = `openshift`, or `kubernetes` depending on if the request is made against the Kubernetes API at `/api`, or the {product-title} API at `/oapi`
+** <git_commit> = The Git commit of the client version (for example, `f034127`)
```

Comment:
These `<replaceable>` items on the left of each bullet in this list could probably get marked up w/ backticks too.